### PR TITLE
Sevak has edited the text

### PR DESCRIPTION
**Pull request by Sevak9699**

### CODE REVIEW

File: song.txt [hunk 1]
```diff
@@ -23,4 +23,6 @@ Enough to make my systems blow
 Welcome to the new age, to the new age
 Welcome to the new age, to the new age
 Whoa, oh, oh, oh, oh, whoa, oh, oh, oh, I'm radioactive, radioactive
-Whoa, oh, oh, oh, oh, whoa, oh, oh, oh, I'm radioactive, radioactive
\ No newline at end of file
+Whoa, oh, oh, oh, oh, whoa, oh, oh, oh, I'm radioactive, radioactive
+
```
Comment (ArturSolomonyan408) [inline song.txt:27]: lava lava

File: song.txt [hunk 2]
```diff
@@ -23,4 +23,6 @@ Enough to make my systems blow
 Welcome to the new age, to the new age
 Welcome to the new age, to the new age
 Whoa, oh, oh, oh, oh, whoa, oh, oh, oh, I'm radioactive, radioactive
-Whoa, oh, oh, oh, oh, whoa, oh, oh, oh, I'm radioactive, radioactive
```
Comment (Amelyan85) [inline song.txt:26]: Good !

File: song.txt [hunk 3]
```diff
@@ -23,4 +23,6 @@ Enough to make my systems blow
...
\ No newline at end of file
+Whoa, oh, oh, oh, oh, whoa, oh, oh, oh, I'm radioactive, radioactive
```
Comment (jgyurjyan) [inline song.txt:26]: nice taste in music!

**Review by jgyurjyan — APPROVED** [review]

**Comment by GayaneEnfenjyan** [comment]
Good!

**Review by Amelyan85 — APPROVED** [review]

**Pull request merged**